.env

```diff
@@ -0,0 +1,2 @@
+NOETL_LOGIN=noetl
+NOETL_PASSWORD=yXByfmFdNYGU
```

Comment:
@Sergo007 is it for development? If nodejs requires `.env` file, why do you assign values to those?
Env variables should come from different source rather than git.

Reply:
@antonputra  is it for development) it need add to git ignore) i will do it now)

Reply:
@antonputra we can add this env to deployment
![image](https://user-images.githubusercontent.com/10484588/48138472-1091a480-e2ad-11e8-8bb3-c8bd0ba94906.png)
but in production we need to do authorization in backend